etcdserver/server.go

```diff
@@ -374,6 +377,8 @@ func NewServer(cfg *ServerConfig) (*EtcdServer, error) {
 			srv.compactor = compactor.NewPeriodic(h, srv.kv, srv)
 			srv.compactor.Run()
 		}
+
+		srv.authStore = auth.NewAuthStore(srv.be)
```

Comment:
Thanks. It seems that the update of `be` isn't notified to related subsystems. Should I add a mechanism for the notification?